lwg-20250314.org

```diff
@@ -0,0 +1,102 @@
+#+options: ':nil *:t -:t ::t <:t H:8 \n:nil ^:t arch:headline author:t
+#+options: broken-links:nil c:nil creator:nil d:(not "LOGBOOK") date:t e:t
+#+options: email:nil expand-links:t f:t inline:t num:t p:nil pri:nil prop:nil
+#+options: stat:t tags:t tasks:t tex:t timestamp:t title:t toc:t todo:t |:t
+#+title: lwg-20250314
+#+date: <2025-03-14 Fri>
+#+author: Steve Downey
+#+email: sdowney@pwyll
+#+language: en
+#+select_tags: export
+#+exclude_tags: noexport
+#+creator: Emacs 30.1.50 (Org mode 9.7.11)
+#+cite_export:
+
+#+options: html-link-use-abs-url:nil html-postamble:auto html-preamble:t
+#+options: html-scripts:t html-style:t html5-fancy:nil tex:t
+#+html_doctype: xhtml-strict
+#+html_container: div
+#+html_content_class: content
+#+description:
+#+keywords:
+#+html_link_home:
+#+html_link_up:
+#+html_mathjax:
+#+html_equation_reference_format: \eqref{%s}
+#+html_head:
+#+html_head_extra:
+#+subtitle:
+#+infojs_opt:
+#+creator: <a href="https://www.gnu.org/software/emacs/">Emacs</a> 30.1.50 (<a href="https://orgmode.org">Org</a> mode 9.7.11)
+#+latex_header:
+
+* Notes
+https://wg21.link/P2988r9 std::optional<T&>
+SD presenting
+[optionalref.iterators]
+TS: p1: “These types” -> “This type”
```

Comment:
Can you please format them?

<img width="1164" alt="image" src="https://github.com/user-attachments/assets/444c548f-316e-4667-998b-c37bcbab66ca" />


L.E. Why do we have duplicated notes?